pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -158,10 +158,66 @@ g {
 nodes_that_need_write = [3, 2, 1, 0]
 
 g {
-	n[0]: &_9  _    => _8 @ bb2[13]: fn exercise_allocator;
-	n[1]: copy n[0] => _7 @ bb2[14]: fn exercise_allocator;
-	n[2]: copy n[1] => _6 @ bb2[16]: fn exercise_allocator;
-	n[3]: copy n[2] => _1 @ bb0[0]:  fn printf;            
+	n[0]:  copy _     => _8  @ bb2[13]:  fn exercise_allocator;     
```

Comment:
all uses of `b"%i\n\x00"` across the program, such as:

```rust
printf(b"%i\n\x00" as *const u8 as *const libc::c_char, ...);
```

Reply:
This sounds like it should be `AddrOfStatic -> Copy`, but `AddrOfStatic` is not implemented yet.

Reply:
agreed